frontend/package.json

```diff
@@ -105,6 +106,7 @@
     "react-hook-form": "^7.33.1",
     "react-i18next": "^11.17.4",
     "react-image-lightbox": "^5.1.4",
+    "react-infinite-scroll-component": "^6.1.0",
```

Comment:
Not sure if you are aware ahooks' useInfiniteScroll :D : https://ahooks.js.org/hooks/use-infinite-scroll

Reply:
yea I know but not easy to use LOL

Reply:
useInfiniteScroll is quite opinionated in how we declare our pagination interface.

The documentation is also quite unclear what exactly they want.

I might be reading it wrongly, but if you used it before can lmk.

Reply:
<img width="426" alt="image" src="https://user-images.githubusercontent.com/24352004/197381087-91274c07-c232-458d-957d-2973f1b3dbf9.png">
<img width="627" alt="image" src="https://user-images.githubusercontent.com/24352004/197381094-b2c7cd4f-629a-4234-8997-ce184ac8f3bf.png">

Offending type. I can't pass ListBetaResponse as the type for this.